com.unity.cinemachine/Runtime/Behaviours/CinemachineFreeLookModifier.cs

```diff
@@ -220,6 +225,52 @@ public override void Reset(CinemachineVirtualCameraBase vcam)
                 CachedComponent.PositionDamping = m_CenterDamping;
         }
     }
+    
+    /// <summary>
+    /// Builtin FreeLook modifier for camera tilt.  Applies a vertical rotation to the camera 
```

Comment:
Inaccurate description (smells like copy pasta).  It's not a tilt, it's a composition adjustment (we already have a tilt).  Also does not do the mod at the end of the pipeline.